Makefile

```diff
@@ -65,3 +64,15 @@ runserver:
 css:
 	$(VENV); gulp sass
 	$(VENV); gulp watch
+
+##### DOCKER INTEGRATION
+##### required docker-compose http://docs.docker.com/compose/install/
+
+docker:
+	docker-compose build
+	docker-compose up -d
+	docker-compose run web src/manage.py migrate
+	docker-compose run web src/manage.py createsuperuser --username=admin --email=admin@example.com
+	docker-compose ps
+	-boot2docker ip
```

Comment:
@intellectronica thats also used here: https://github.com/divio/django-cms-demo/blob/master/Makefile#L29

when you might have a dirty tree, I want the command to fail silently instead of cancelling the following commands